### PR TITLE
claim submitted page

### PR DESCRIPTION
**Pull request by kamilewandowski**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-4406

### Change description ###
add claim submitted page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/views/features/claim/claim-submitted.njk [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      {{ govukPanel({
+        titleText: t('PAGES.CLAIM_SUBMITTED.PAGE_TITLE'),
+        html: panelContent
+      }) }}
+    </div>
+      {% if helpWithFee %}
+        <div class="govuk-body">
+          <h2>{{ t('PAGES.SUBMIT_CONFIRMATION.WHAT_HAPPENS_NEXT') }}</h2>
+          <p>
+          Your claim will be issued once your Help With Fees application has been confirmed.
```
Comment (laperla11) [inline src/main/views/features/claim/claim-submitted.njk:32]: pls use translation keys

File: src/main/views/features/claim/claim-submitted.njk [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          We'll email you within 5 days to confirm this.
+          </p>
+        </div>
+      {% else %}
+        <div class="govuk-body">
+          <h2>{{ t('PAGES.SUBMIT_CONFIRMATION.WHAT_HAPPENS_NEXT') }}</h2>
+          <p>
+          {{ defendantFullName }} has until {{ defendantResponseLimit }}
```
Comment (laperla11) [inline src/main/views/features/claim/claim-submitted.njk:40]: same here

File: src/main/routes/features/claim/claimSubmittedController.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import {NextFunction, Router} from 'express';
+import {CASE_DOCUMENT_DOWNLOAD_URL, CLAIM_SUBMITTED_URL} from '../../urls';
+import {getClaimById} from 'modules/utilityService';
+import {DocumentUri} from 'models/document/documentType';
+import {YesNo} from 'form/models/yesNo';
+
+const claimSubmittedView = 'features/claim/claim-submitted';
+const claimSubmittedController = Router();
+
+claimSubmittedController.get(CLAIM_SUBMITTED_URL, async (req, res, next: NextFunction) => {
```
Comment (laperla11) [inline src/main/routes/features/claim/claimSubmittedController.ts:10]: use  this url : `CLAIM_CONFIRMATION_URL` for your page which is already defined and used for redirecting your page from previous page

**Review by laperla11 — CHANGES_REQUESTED** [review]

File: src/main/modules/i18n/locales/en.json [hunk 4]
```diff
@@ -593,6 +593,10 @@
         "TITLE": "When are you claiming interest from?"
       }
     },
+    "CLAIM_SUBMITTED": {
```
Comment (laarmada) [inline src/main/modules/i18n/locales/en.json:596]: where is the tranlation in the cy.json

File: src/main/routes/features/claim/claimSubmittedController.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  try {
+    const claimId = req.params.id;
+    const claim = await getClaimById(claimId, req);
+    if(!claim.isEmpty()) {
+      const claimNumber = claim.legacyCaseReference;
+      const downloadHref = CASE_DOCUMENT_DOWNLOAD_URL.replace(':id', claimId).replace(':documentType', DocumentUri.SEALED_CLAIM);
+      const defendantFullName = claim.getDefendantFullName();
+      const defendantResponseLimit = claim.responseDeadline?.calculatedResponseDeadline;
+      const helpWithFee = claim.claimDetails?.helpWithFees?.option == YesNo.YES;
```
Comment (laarmada) [inline src/main/routes/features/claim/claimSubmittedController.ts:19]: maybe we have a function for that is used in several pages. claim.claimDetails?.helpWithFees?.option == YesNo.YES;

**Review by laarmada — CHANGES_REQUESTED** [review]

File: src/main/routes/features/claim/claimSubmittedController.ts (hunk 5 above)
Comment (kamilewandowski) [reply src/main/routes/features/claim/claimSubmittedController.ts:19]: I did not find a function like this. I can create one.

**Review by laarmada — APPROVED** [review]

**Review by laperla11 — APPROVED** [review]

**Pull request closed without merging**